test/kickstart-templates/kickstart-offline.ks.template

```diff
@@ -0,0 +1,108 @@
+#
+# For example, a 20GB disk would be partitioned in the following way:
+#
+# NAME          MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
+# sda             8:0    0  20G  0 disk
+# ├─sda1          8:1    0 200M  0 part /boot/efi
+# ├─sda1          8:1    0 800M  0 part /boot
+# └─sda2          8:2    0  19G  0 part
+#  └─rhel-root  253:0    0  10G  0 lvm  /sysroot
+#
+zerombr
+clearpart --all --initlabel
+part /boot/efi --fstype=efi --size=200
+part /boot --fstype=xfs --asprimary --size=800
+# Uncomment this line to add a SWAP partition of the recommended size
+#part swap --fstype=swap --recommended
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=REPLACE_LVM_SYSROOT_SIZE --name=root
+
+# Lock root user account
+rootpw --lock
+
+# Configure non-ostree filesystem
+liveimg --url file:///run/install/repo/liveimg.tar.gz
+
+%post --log=/var/log/anaconda/post-install.log --erroronfail
+
+# Set the configuration of MicroShift to include subjectAltNames
+mkdir -p /etc/microshift
+cat - >/etc/microshift/config.yaml <<EOF
+apiServer:
+  subjectAltNames:
+EOF
+
+DEFAULT_ROUTE_IPS="$(ip route show default 2>/dev/null | awk '{print $9}')"
+if [ -n "${DEFAULT_ROUTE_IPS}" ] ; then
+    for r in ${DEFAULT_ROUTE_IPS} ; do
+        echo "    - ${r}" >>/etc/microshift/config.yaml
+    done
+else
+    MAIN_HOST_IP="$(hostname -I | awk '{print $1}')"
+    echo "    - ${MAIN_HOST_IP}" >>/etc/microshift/config.yaml
+fi
+if [ -n "REPLACE_PUBLIC_IP" ]; then
+    echo "    - REPLACE_PUBLIC_IP" >>/etc/microshift/config.yaml
+fi
+
+# The pull secret is mandatory for MicroShift builds on top of OpenShift, but not OKD
+# The /etc/crio/crio.conf.d/microshift.conf references the /etc/crio/openshift-pull-secret file
+mkdir -p /etc/crio
+cat > /etc/crio/openshift-pull-secret <<EOF
+REPLACE_PULL_SECRET
+EOF
+chmod 600 /etc/crio/openshift-pull-secret
+
+# Create a default redhat user, allowing it to run sudo commands without password
+useradd -m -d /home/redhat -p \$5\$XDVQ6DxT8S5YWLV7\$8f2om5JfjK56v9ofUkUAwZXTxJl3Sqnc9yPnza4xoJ0 redhat
+echo -e 'redhat\tALL=(ALL)\tNOPASSWD: ALL' > /etc/sudoers.d/microshift
+
+# # Add authorized ssh keys
+mkdir -m 700 /home/redhat/.ssh
+cat >> /home/redhat/.ssh/authorized_keys <<EOF
+REPLACE_REDHAT_AUTHORIZED_KEYS
+EOF
+chmod 600 /home/redhat/.ssh/authorized_keys
+
+# Make sure redhat user directory contents ownership is correct
+chown -R redhat:redhat /home/redhat/
+
+# Make the KUBECONFIG from MicroShift directly available for the root user
+echo -e 'export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig' >> /root/.profile
+
+# Configure systemd journal service to persist logs between boots and limit their size to 1G
+sudo mkdir -p /etc/systemd/journald.conf.d
+cat > /etc/systemd/journald.conf.d/microshift.conf <<EOF
+[Journal]
+Storage=persistent
+SystemMaxUse=1G
+RuntimeMaxUse=1G
+EOF
+
+# Make sure all the Ethernet network interfaces are connected automatically
```

Comment:
It seems like we want to either remove this or change it to not enable any interfaces, right?

Reply:
There are no eth interfaces, so this code is mute. 
Removing.